ports.mk

```diff
@@ -92,11 +92,139 @@ endif
 	cp $(srcdir)/dists/iphone/icon-72.png $(bundle_name)/
 	cp $(srcdir)/dists/iphone/Default.png $(bundle_name)/
 
+ios7bundle: ios7
+	mkdir -p $(bundle_name)
+	awk 'BEGIN {s=0}\
```

Comment:
The reason why I had to edit the file here is related to how Xcode handles this file, due to the usage of Asset Catalog. If we put the keys directly in the Info.plist file, these are removed by Xcode each time we compile the project, and it changes the file on disk.